src/diffusers/models/attention_processor.py

```diff
@@ -2750,6 +2763,117 @@ def __call__(
         return hidden_states
 
 
+class XLAFlashAttnProcessor2_0:
```

Comment:
So, this will be _automatically_ used when using the compatible models under an XLA environment, right?

Reply:
Yes, AttnProcessor2_0 will be replaced with XLAFlashAttnProcessor2_0 if XLA version condition satisfied.